src/averageAges.js

```diff
@@ -15,13 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = filterPeopleBySex(people, 'm');
+
+  const foundMan = century
+    ? men.filter(man => Math.ceil(man.died / 100) === century)
+    : men;
+
+  return calculateAverageAge(foundMan);
 }
 
+const filterPeopleBySex = (people, sex) => {
+  return people.filter(person => person.sex === sex);
+};
```

Comment:
I think, it would be better to move this helper function to the end of the file )